.github/workflows/arrow-flight-tests.yml

```diff
@@ -48,6 +48,142 @@ jobs:
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl ${{ matrix.modules }}
 
-      # Run Maven tests for the target module
+      # Run Maven tests for the target module, excluding native tests
       - name: Maven Tests
-        run: ./mvnw test ${MAVEN_TEST} -pl ${{ matrix.modules }}
+        run: ./mvnw test ${MAVEN_TEST} -pl ${{ matrix.modules }} -Dtest="*,!TestArrowFlightNativeQueries"
+
+  prestocpp-linux-build-for-test:
```

Comment:
@majetideepak @czentgr I took most of this from `prestocpp-linux-build-and-unit-test.yml`, but it seems to be timing out waiting for the worker nodes to come up and I'm not sure why. Do you have any ideas to troubleshoot?